Tasks/Common/azure-blobstorage-artifactProvider-v2/azureBlobStorageProvider.ts

```diff
@@ -155,6 +162,17 @@ export class AzureBlobProvider implements models.IArtifactProvider {
         return artifactItems;
     }
 
+     /**
+     * Get file name from full path to this file.
+     * @returns {string} File name without folders
+     * @param {string} path Path to file
+     */
+    private _getFileName(path: string): string {
```

Comment:
Might be static

Reply:
Seems like it's possible to use [path.basename](https://stackoverflow.com/questions/19811541/nodejs-get-file-name-from-absolute-path) instead?

Reply:
> Seems like it's possible to use [path.basename](https://stackoverflow.com/questions/19811541/nodejs-get-file-name-from-absolute-path) instead?

Thank you for your suggestion! It makes sense.